filament/include/filament/ColorGrading.h

```diff
@@ -152,6 +154,18 @@ class UTILS_PUBLIC ColorGrading : public FilamentAPI {
          */
         Builder& toneMapping(ToneMapping toneMapping) noexcept;
 
+        /**
+         * Adjusts the exposure of this image. The exposure is specified in stops:
```

Comment:
Maybe we can have a small explanation of how that differs from the camera exposure and how the two interact?
e.g. I could imagine scenarios where people do it wrong and adjust this exposure usual is the camera's?

Reply:
It does the same thing except because it's in the color grading pass it's applied post screen space effects. I don't think it makes sense to try and compare it to the camera.

Reply:
Done.